marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/CreateGranularPrivilegesCommand.java

```diff
@@ -228,11 +234,17 @@ public void undo(CommandContext context) {
         ROLES_THAT_CAN_BE_INHERITED.forEach(roleName -> {
             // We expect each role to translate to a role; otherwise, an error should be thrown
             String roleId = existingRoles.getIdForNameOrId(roleName);
-            Privilege priv = newPrivilege("data-role-inherit-" + roleId, "data-hub-security-admin");
+            Privilege priv = newPrivilege("data-role-inherit-" + roleId, "data-hub-role-inheritor");
             priv.setAction("http://marklogic.com/xdmp/privileges/role/inherit/" + roleId);
             granularPrivilegeMap.put(priv.getAction(), priv);
         });
 
+        //This privilege should be assigned to "dh-security-admin" only so that it can create custom roles that have "dh-amp-manager"
```

Comment:
I don't think this needs to be done - i.e. this privilege can go to the role-inheritor role. Only a dhsa can create custom roles. But a dh-amp-manager should be able to create an amp that inherits dh-amp-manager.